src/coreclr/inc/loaderheap.h

```diff
@@ -158,7 +158,7 @@ struct LoaderHeapEvent;
 // When an interleaved LoaderHeap is constructed, this is the interleaving size
 inline UINT32 GetStubCodePageSize()
 {
-#if defined(TARGET_ARM64) && defined(TARGET_UNIX)
+#if (defined(TARGET_ARM64) && defined(TARGET_UNIX)) || defined(TARGET_WASM)
```

Comment:
What's the page size that this method computes on wasm? It is surprising that 16kB is not enough.

Reply:
Is that part of the wasm spec, or is it just a hardcoded constant in emscripten mmap simulator?

Reply:
> Is that part of the wasm spec, or is it just a hardcoded constant in emscripten mmap simulator?
> As far as I know it's to spec.

It is also kind of both - the value has changed at least two times in Emscripten's history (4k -> 16k -> 64k).